### ISSUE TITLE
Chains not being created - fail2ban v0.10.0a2 - CloudLinux 6.9

### ISSUE THREAD

**Issue by Mattz0r** [fail2ban/fail2ban#1755]
### Environment:

_Fill out and check (`[x]`) the boxes which apply. If your Fail2Ban version is outdated, 
and you can't verify that the issue persists in the recent release, better seek support 
from the distribution you obtained Fail2Ban from_

- Fail2Ban version: v0.10.0a2
- OS, including release name/version: Cloudlinux 6.9 (SELinux disabled)
- [ ] Fail2Ban installed via OS/distribution mechanisms
- [x] You have not applied any additional foreign patches to the codebase
- [ ] Some customizations were done to the configuration (provide details below is so)

### The issue:

After installing the latest version of fail2ban on this server (no previous versions installed), I imported my existing configurations which work perfectly fine with fail2ban 0.9 on other installations. I am able to start the service, but for some reason the chains are not being created within iptables.

#### Steps to reproduce

service fail2ban restart

#### Expected behavior

The below chains being created in iptables:

f2b-proftpd
apache-wp-login
apache-wp-xmlrpc
apache-joomla-login
admin-banned-ftp
admin-banned-http

#### Observed behavior

Chains not created.

### Configuration, dump and another helpful excerpts

```
# Fail2Ban main configuration file
#
# Comments: use '#' for comment lines and ';' for inline comments
#
# Changes:  in most of the cases you should not modify this
#           file, but provide customizations in fail2ban.local file, e.g.:
#
# [Definition]
# loglevel = 4
#

[Definition]

# Option:  loglevel
# Notes.:  Set the log level output.
#          1 = ERROR
#          2 = WARN
#          3 = INFO
#          4 = DEBUG
# Values:  NUM  Default:  3
#
loglevel = DEBUG

# Option:  logtarget
# Notes.:  Set the log target. This could be a file, SYSLOG, STDERR or STDOUT.
#          Only one log target can be specified.
# Values:  STDOUT STDERR SYSLOG file  Default:  /var/log/fail2ban.log
#
logtarget = /var/log/fail2ban.log

# Option: socket
# Notes.: Set the socket file. This is used to communicate with the daemon. Do
#         not remove this file when Fail2ban runs. It will not be possible to
#         communicate with the server afterwards.
# Values: FILE  Default:  /var/run/fail2ban/fail2ban.sock
#
socket = /var/run/fail2ban/fail2ban.sock

pidfile = /var/run/fail2ban/fail2ban.pid
syslogsocket = auto
dbfile = /var/lib/fail2ban/fail2ban.sqlite3
dbpurgeage = 86400
```

# jail.local file
```
[DEFAULT]
backend = auto
ignoreip = xxx.xxx.xxx.xxx/24

[apache-wp-login]
enabled = true
port = http,https
filter = apache-wp-login
action = iptables-multiport[name=wp-login, port="http,https", protocol=tcp]
         combined_tracker
logpath = /hsphere/shared/apache2/logs/combined_log
maxretry = 30
bantime  = 86400
findtime = 1800

[apache-wp-xmlrpc]
enabled = true
port = http,https
filter = apache-wp-xmlrpc
action = iptables-multiport[name=wp-xmlrpc, port="http,https", protocol=tcp]
         combined_tracker
logpath = /hsphere/shared/apache2/logs/combined_log
maxretry = 30
bantime  = 86400
findtime = 1800

[apache-joomla-login]
enabled = yes
port = http,https
filter = apache-joomla-login
action = iptables-multiport[name=joomla, port="http,https", protocol=tcp]
         combined_tracker
logpath = /hsphere/shared/apache2/logs/combined_log
maxretry = 30
bantime  = 86400
findtime = 1800

[proftpd]
enabled = yes
port     = ftp,ftp-data,ftps,ftps-data
filter = proftpd
action = iptables-multiport[name=proftpd, port="ftp,ftp-data,ftps,ftps-data", protocol=tcp]
         auth_tracker
logpath = /hsphere/local/var/proftpd/auth.log
maxretry = 30
bantime  = 86400
findtime = 1200

[admin-banned-http]
enabled  = true
port = http,https
action   = iptables-multiport[name=admin-banned-http, port="http,https", protocol=tcp]
logpath  = /var/log/adminbanned
maxretry = 5
bantime = 86400
findtime = 1200

[admin-banned-ftp]
enabled  = true
port = ftp,ftp-data,ftps,ftps-data
action   = iptables-multiport[name=admin-banned-ftp, port="ftp,ftp-data,ftps,ftps-data", protocol=tcp]
logpath  = /var/log/adminbanned
maxretry = 5
bantime = 86400
findtime = 1200
```
# iptables -L -n output after starting fail2ban
```
Chain INPUT (policy ACCEPT)
target     prot opt source               destination

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination
```
#### Any customizations done to /etc/fail2ban/ configuration
```
```

#### Relevant parts of /var/log/fail2ban.log file:
_preferably obtained while running fail2ban with `loglevel = 4`_

```
2017-04-19 13:22:27,184 fail2ban.transmitter    [132255]: WARNING Command ['set', 'admin-banned-ftp', 'addfailregex', "''"] has failed. Received RegexException("No failure-id group in ''''",)
Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/fail2ban/server/transmitter.py", line 57, in proceed
    ret = self.__commandHandler(command)
  File "/usr/lib/python2.6/site-packages/fail2ban/server/transmitter.py", line 127, in __commandHandler
    return self.__commandSet(command[1:])
  File "/usr/lib/python2.6/site-packages/fail2ban/server/transmitter.py", line 234, in __commandSet
    self.__server.addFailRegex(name, value, multiple=multiple)
  File "/usr/lib/python2.6/site-packages/fail2ban/server/server.py", line 401, in addFailRegex
    flt.addFailRegex(value)
  File "/usr/lib/python2.6/site-packages/fail2ban/server/filter.py", line 171, in addFailRegex
    raise e
RegexException: No failure-id group in ''''
```

```
2017-04-19 13:22:27,203 fail2ban.jail           [132255]: DEBUG   Starting jail 'proftpd'
2017-04-19 13:22:27,204 fail2ban.filterpoll     [132255]: DEBUG   /hsphere/local/var/proftpd/auth.log has been modified
2017-04-19 13:22:27,204 fail2ban.filter         [132255]: DEBUG   Seek to find time 1492603947.02 (2017-04-19 13:12:27), file size 178
2017-04-19 13:22:27,204 fail2ban.filter         [132255]: DEBUG   Position -1 from 178, found time None () within 0 seeks
2017-04-19 13:22:27,206 fail2ban.jail           [132255]: INFO    Jail 'proftpd' started
2017-04-19 13:22:27,206 fail2ban.jail           [132255]: DEBUG   Starting jail 'apache-wp-login'
2017-04-19 13:22:27,207 fail2ban.filterpoll     [132255]: DEBUG   /hsphere/shared/apache2/logs/combined_log has been modified
2017-04-19 13:22:27,207 fail2ban.filter         [132255]: DEBUG   Seek to find time 1492603947.04 (2017-04-19 13:12:27), file size 840
2017-04-19 13:22:27,207 fail2ban.filter         [132255]: DEBUG   Position -1 from 840, found time None () within 0 seeks
2017-04-19 13:22:27,208 fail2ban.jail           [132255]: INFO    Jail 'apache-wp-login' started
2017-04-19 13:22:27,209 fail2ban.jail           [132255]: DEBUG   Starting jail 'apache-wp-xmlrpc'
2017-04-19 13:22:27,209 fail2ban.filterpoll     [132255]: DEBUG   /hsphere/shared/apache2/logs/combined_log has been modified
2017-04-19 13:22:27,209 fail2ban.filter         [132255]: DEBUG   Seek to find time 1492603947.06 (2017-04-19 13:12:27), file size 840
2017-04-19 13:22:27,210 fail2ban.filter         [132255]: DEBUG   Position -1 from 840, found time None () within 0 seeks
2017-04-19 13:22:27,211 fail2ban.jail           [132255]: INFO    Jail 'apache-wp-xmlrpc' started
2017-04-19 13:22:27,211 fail2ban.jail           [132255]: DEBUG   Starting jail 'apache-joomla-login'
2017-04-19 13:22:27,212 fail2ban.filterpoll     [132255]: DEBUG   /hsphere/shared/apache2/logs/combined_log has been modified
2017-04-19 13:22:27,212 fail2ban.filter         [132255]: DEBUG   Seek to find time 1492603947.09 (2017-04-19 13:12:27), file size 840
2017-04-19 13:22:27,212 fail2ban.filter         [132255]: DEBUG   Position -1 from 840, found time None () within 0 seeks
2017-04-19 13:22:27,213 fail2ban.jail           [132255]: INFO    Jail 'apache-joomla-login' started
2017-04-19 13:22:27,213 fail2ban.jail           [132255]: DEBUG   Starting jail 'admin-banned-http'
2017-04-19 13:22:27,214 fail2ban.filterpoll     [132255]: DEBUG   /var/log/adminbanned has been modified
2017-04-19 13:22:27,214 fail2ban.filter         [132255]: DEBUG   Seek to find time 1492603947.17 (2017-04-19 13:12:27), file size 15
2017-04-19 13:22:27,214 fail2ban.filter         [132255]: DEBUG   Position -1 from 15, found time None () within 0 seeks
2017-04-19 13:22:27,215 fail2ban.jail           [132255]: INFO    Jail 'admin-banned-http' started
2017-04-19 13:22:27,216 fail2ban.jail           [132255]: DEBUG   Starting jail 'admin-banned-ftp'
2017-04-19 13:22:27,216 fail2ban.filterpoll     [132255]: DEBUG   /var/log/adminbanned has been modified
2017-04-19 13:22:27,216 fail2ban.filter         [132255]: DEBUG   Seek to find time 1492603947.19 (2017-04-19 13:12:27), file size 15
2017-04-19 13:22:27,217 fail2ban.filter         [132255]: DEBUG   Position -1 from 15, found time None () within 0 seeks
2017-04-19 13:22:27,217 fail2ban.jail           [132255]: INFO    Jail 'admin-banned-ftp' started
```

#### Relevant lines from monitored log files in question:

```
```

**Comment by sebres** [comment]
1. jail `admin-banned-ftp` seems to have no filter (you should create a filter file `filter.d/admin-banned-ftp.local` or specify another filter file via `filter = my-admin-banned` that references `filter.d/my-admin-banned.conf` or `filter.d/my-admin-banned.local`).
2. Otherwise (if filter however exists), it may have not correct failregex entry...
3. Try to dump your configuration for some jail and post the result here, e. g. for `proftpd` and `admin-banned-ftp`:
```bash
fail2ban-client -d | grep proftpd
fail2ban-client -d | grep admin-banned-ftp
```

**Comment by Mattz0r** [comment]
`[root@cltest ~]# fail2ban-client -d | grep proftpd
['add', 'proftpd', 'auto']
['set', 'proftpd', 'addfailregex', '\\S+\\s\\[\\d+\\]\\s<HOST>\\s\\[.+\\]\\s\\"PASS \\(hidden\\)\\" 530']
['set', 'proftpd', 'usedns', 'warn']
['set', 'proftpd', 'addlogpath', '/hsphere/local/var/proftpd/auth.log', 'head']
['set', 'proftpd', 'maxretry', 30]
['set', 'proftpd', 'addignoreip', 'xx.xx.xx.xx/24']
['set', 'proftpd', 'logencoding', 'auto']
['set', 'proftpd', 'bantime', '86400']
['set', 'proftpd', 'ignorecommand', '']
['set', 'proftpd', 'findtime', '1200']
['set', 'proftpd', 'addaction', 'iptables-multiport']
['multi-set', 'proftpd', 'action', 'iptables-multiport', [['actionunban', '<iptables> -D f2b-proftpd -s <ip> -j <blocktype>'], ['actionflush', '<iptables> -F f2b-proftpd'], ['actionstop', '<iptables> -D INPUT -p tcp -m multiport --dports ftp,ftp-data,ftps,ftps-data -j f2b-proftpd\n<iptables> -F f2b-proftpd\n<iptables> -X f2b-proftpd'], ['actionstart', '<iptables> -N f2b-proftpd\n<iptables> -A f2b-proftpd -j RETURN\n<iptables> -I INPUT -p tcp -m multiport --dports ftp,ftp-data,ftps,ftps-data -j f2b-proftpd'], ['actionban', '<iptables> -I f2b-proftpd 1 -s <ip> -j <blocktype>'], ['actioncheck', "<iptables> -n -L INPUT | grep -q 'f2b-proftpd[ \\t]'"], ['iptables', 'iptables <lockingopt>'], ['lockingopt', '-w'], ['known/lockingopt', '-w'], ['blocktype?family=inet6', 'REJECT --reject-with icmp6-port-unreachable'], ['protocol', 'tcp'], ['name', 'proftpd'], ['chain', 'INPUT'], ['known/blocktype?family=inet6', 'REJECT --reject-with icmp6-port-unreachable'], ['known/protocol', 'tcp'], ['known/port', 'ssh'], ['known/returntype', 'RETURN'], ['known/iptables', 'iptables <lockingopt>'], ['known/chain', 'INPUT'], ['returntype', 'RETURN'], ['known/name', 'default'], ['known/blocktype', 'REJECT --reject-with icmp-port-unreachable'], ['iptables?family=inet6', 'ip6tables <lockingopt>'], ['known/iptables?family=inet6', 'ip6tables <lockingopt>'], ['blocktype', 'REJECT --reject-with icmp-port-unreachable'], ['actname', 'iptables-multiport'], ['port', 'ftp,ftp-data,ftps,ftps-data']]]
['set', 'proftpd', 'addaction', 'auth_tracker']
['multi-set', 'proftpd', 'action', 'auth_tracker', [['actioncheck', ''], ['actionban', 'echo "$( date \'+%Y-%m-%d %T\' ) <ip> BANNED" >> /hsphere/local/var/proftpd/auth.log'], ['actionstop', ''], ['actionstart', ''], ['actionunban', 'echo "$( date \'+%Y-%m-%d %T\' ) <ip> UNBANNED" >> /hsphere/local/var/proftpd/auth.log'], ['actname', 'auth_tracker']]]
['start', 'proftpd']
`
`[root@cltest ~]# fail2ban-client -d | grep admin-banned-ftp
['add', 'admin-banned-ftp', 'auto']
['set', 'admin-banned-ftp', 'addfailregex', "''"]
['set', 'admin-banned-ftp', 'usedns', 'warn']
['set', 'admin-banned-ftp', 'addlogpath', '/var/log/adminbanned', 'head']
['set', 'admin-banned-ftp', 'maxretry', 5]
['set', 'admin-banned-ftp', 'addignoreip', 'xx.xxx.xx.xx/24']
['set', 'admin-banned-ftp', 'logencoding', 'auto']
['set', 'admin-banned-ftp', 'bantime', '86400']
['set', 'admin-banned-ftp', 'ignorecommand', '']
['set', 'admin-banned-ftp', 'findtime', '1200']
['set', 'admin-banned-ftp', 'addaction', 'iptables-multiport']
['multi-set', 'admin-banned-ftp', 'action', 'iptables-multiport', [['actionunban', '<iptables> -D f2b-admin-banned-ftp -s <ip> -j <blocktype>'], ['actionflush', '<iptables> -F f2b-admin-banned-ftp'], ['actionstop', '<iptables> -D INPUT -p tcp -m multiport --dports ftp,ftp-data,ftps,ftps-data -j f2b-admin-banned-ftp\n<iptables> -F f2b-admin-banned-ftp\n<iptables> -X f2b-admin-banned-ftp'], ['actionstart', '<iptables> -N f2b-admin-banned-ftp\n<iptables> -A f2b-admin-banned-ftp -j RETURN\n<iptables> -I INPUT -p tcp -m multiport --dports ftp,ftp-data,ftps,ftps-data -j f2b-admin-banned-ftp'], ['actionban', '<iptables> -I f2b-admin-banned-ftp 1 -s <ip> -j <blocktype>'], ['actioncheck', "<iptables> -n -L INPUT | grep -q 'f2b-admin-banned-ftp[ \\t]'"], ['iptables', 'iptables <lockingopt>'], ['lockingopt', '-w'], ['known/lockingopt', '-w'], ['blocktype?family=inet6', 'REJECT --reject-with icmp6-port-unreachable'], ['protocol', 'tcp'], ['name', 'admin-banned-ftp'], ['chain', 'INPUT'], ['known/blocktype?family=inet6', 'REJECT --reject-with icmp6-port-unreachable'], ['known/protocol', 'tcp'], ['known/port', 'ssh'], ['known/returntype', 'RETURN'], ['known/iptables', 'iptables <lockingopt>'], ['known/chain', 'INPUT'], ['returntype', 'RETURN'], ['known/name', 'default'], ['known/blocktype', 'REJECT --reject-with icmp-port-unreachable'], ['iptables?family=inet6', 'ip6tables <lockingopt>'], ['known/iptables?family=inet6', 'ip6tables <lockingopt>'], ['blocktype', 'REJECT --reject-with icmp-port-unreachable'], ['actname', 'iptables-multiport'], ['port', 'ftp,ftp-data,ftps,ftps-data']]]
['start', 'admin-banned-ftp']
`

I created the admin-banned filters, so these can be created manually, using fail2ban-client set add admin-banned-http banip x.x.x.x - Just keeps them separate from the automated rules.

**Comment by sebres** [comment]
The iptables actions in v.0.10 create chains on demand (i. e. by first ban of some IP, corresponding its IP-family), see #1742 for more info.

If you don't want it, use `iptables-multiport[..., actionstart_on_demand=false]` in your `jail.local` resp. `actionstart_on_demand = false` in definition section of your `action.d/iptables-multiport.local`.

**Comment by Mattz0r** [comment]
Understood.

If I evaluate this further, and attempt to ban an IP, the chain is still not created due to errors below:

```
2017-04-20 11:13:13,680 fail2ban.filter         [284616]: WARNING Requested to manually ban an ignored IP 213.x.x.x. User knows best. Proceeding to ban it.
2017-04-20 11:13:13,680 fail2ban.failmanager    [284616]: DEBUG   Total # of detected failures: 2. Current failures from 1 IPs (IP:count): 213.x.x.x:5
2017-04-20 11:13:13,761 fail2ban.actions        [284616]: NOTICE  [admin-banned-ftp] Ban 213.x.x.x
2017-04-20 11:13:13,761 fail2ban.action         [284616]: DEBUG   iptables -w -N f2b-default
iptables -w -A f2b-default -j RETURN
iptables -w -I INPUT -p tcp -m multiport --dports ssh -j f2b-default
2017-04-20 11:13:13,774 fail2ban.utils          [284616]: Level 39 7f1f1aee7df0 -- exec: iptables -w -N f2b-default
iptables -w -A f2b-default -j RETURN
iptables -w -I INPUT -p tcp -m multiport --dports ssh -j f2b-default
2017-04-20 11:13:13,775 fail2ban.utils          [284616]: ERROR   7f1f1aee7df0 -- stderr: "iptables v1.4.7: option `-w' requires an argument"
2017-04-20 11:13:13,775 fail2ban.utils          [284616]: ERROR   7f1f1aee7df0 -- stderr: "Try `iptables -h' or 'iptables --help' for more information."
2017-04-20 11:13:13,775 fail2ban.utils          [284616]: ERROR   7f1f1aee7df0 -- stderr: "iptables v1.4.7: option `-w' requires an argument"
2017-04-20 11:13:13,775 fail2ban.utils          [284616]: ERROR   7f1f1aee7df0 -- stderr: "Try `iptables -h' or 'iptables --help' for more information."
2017-04-20 11:13:13,775 fail2ban.utils          [284616]: ERROR   7f1f1aee7df0 -- stderr: "iptables v1.4.7: option `-w' requires an argument"
2017-04-20 11:13:13,775 fail2ban.utils          [284616]: ERROR   7f1f1aee7df0 -- stderr: "Try `iptables -h' or 'iptables --help' for more information."
2017-04-20 11:13:13,775 fail2ban.utils          [284616]: ERROR   7f1f1aee7df0 -- returned 2
2017-04-20 11:13:13,776 fail2ban.actions        [284616]: ERROR   Failed to execute ban jail 'admin-banned-ftp' action 'iptables-multiport' info 'ActionInfo({'ipfailures': 1, 'ip-rev': 'x.x.x.213.', 'family': 'inet4', 'ipmatches': '', 'matches': '', 'ip': 213.x.x.x, 'ipjailmatches': '', 'ipjailfailures': 1, 'F-*': {'matches': [], 'failures': 1}, 'fid': 213.x.x.x, 'time': 1492683193.6807899, 'failures': 1, 'restored': 0, 'ip-host': 'snip-213.snip'})': Error starting action Jail('admin-banned-ftp')/iptables-multiport
Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/fail2ban/server/actions.py", line 400, in __checkBan
    action.ban(aInfo)
  File "/usr/lib/python2.6/site-packages/fail2ban/server/action.py", line 424, in ban
    self.start(family)
  File "/usr/lib/python2.6/site-packages/fail2ban/server/action.py", line 406, in start
    return self._executeOperation('<actionstart>', 'starting', family=family)
  File "/usr/lib/python2.6/site-packages/fail2ban/server/action.py", line 375, in _executeOperation
    raise RuntimeError("Error %s action %s/%s" % (operation, self._jail, self._name,))
RuntimeError: Error starting action Jail('admin-banned-ftp')/iptables-multiport
2017-04-20 11:13:13,779 fail2ban.actions        [284616]: DEBUG   Banned 1 / 2, 1 ticket(s) in 'admin-banned-ftp'
```

Is this something I've done wrong?

**Comment by sebres** [comment]
> Is this something I've done wrong?

Yes, the option `-w` does not supported by your system or (e. g. if yet SELinux) requires an argument (boolean? yes/no? no idea which one, see `iptables --help`).

This option is controllable in fail2ban by [lockingopt](https://github.com/fail2ban/fail2ban/blob/0.10/config/action.d/iptables-common.conf#L71)
So e. g. you can set it your jail.local like here:
```
[DEFAULT]
banaction = iptables-multiport[lockingopt=""]
_action = %(banaction)s[name=%(__name__)s, bantime="%(bantime)s", port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]

[jail1]
port = ...
action = %(_action)s[extra_option_for_jail1=...]

[jail1]
port = ...
action = %(_action)s[extra_option_for_jail2=...]
```
Or direct per jail, for example:
```
[proftpd]
# remove "-w" option:
action = iptables-multiport[lockingopt="", name=proftpd, port="ftp,ftp-data,ftps,ftps-data", protocol=tcp]
         auth_tracker

[admin-banned-http]
# if "on" would be right argument for "-w" (see iptables --help):
action   = iptables-multiport[lockingopt="-w on" name=admin-banned-http, port="http,https", protocol=tcp]
```
Or just disable (resp. overwrite) it once in your custom action, like suggested in https://github.com/fail2ban/fail2ban/issues/1300#issuecomment-172088893

**Comment by sebres** [comment]
BTW run constantly with debug level is not so good idea, at least it would hugely grow your fail2ban log-file... 😄

**Comment by Mattz0r** [comment]
Haha, yeah I get that :) but until this is resolved... 

Removing the -w option has sorta "fixed it", but for some reason instead of chains being created with the relevant names, they are using f2b-default

```
[root@cltest fail2ban]# fail2ban-client set admin-banned-ftp banip 213.x.x.x
213.x.x.x
[root@cltest fail2ban]# iptables -L -n |grep admin-banned-ftp
[root@cltest fail2ban]# iptables -L -n |grep f2b-default -A4 -B3

Chain f2b-default (1 references)
target     prot opt source               destination
REJECT     all  --  213.x.x.x         0.0.0.0/0           reject-with icmp-port-unreachable
RETURN     all  --  0.0.0.0/0            0.0.0.0/0
```

**Comment by sebres** [comment]
Bad thing, can you please provide relevant config-lines and where it is (how you "fixed" it)?
Additionally interesting `fail2ban-client -d | grep admin-banned-ftp`...

The chain is an interpolation of `f2b-<name>`, where `name` will be normally supplied as action parameter (usually a section name e. g. via `%(__name__)s`, so name of jail). 
So either you misconfigured somewhere ... or indeed you've found a bug 😄

**Comment by Mattz0r** [comment]
I'll just keep quiet haha,

I broke it myself :) we'll just leave it at that and I'll walk away quietly with my head held down lol!

But on a side note, I somehow managed to put [Init?family=inet4] in one of the sections, because... science. I have removed the bits about [Init?family=inet6] because I don't use ipv6 at all..

Hey preseto (after removing ?family=inet4)

```
[root@cltest fail2ban]# fail2ban-client set admin-banned-ftp banip 213.x.x.x

[root@cltest fail2ban]# iptables -L -n |grep admin-banned-ftp -A2

Chain f2b-admin-banned-ftp (1 references)
target     prot opt source               destination
REJECT     all  --  213.x.x.x         0.0.0.0/0           reject-with icmp-port-unreachable
```

That's what happens when you take things into your own little world to try and fix things that aren't actually broken lol.

Sorry for partially wasting your time \o/
